server/models/recipe.py

```diff
@@ -0,0 +1,155 @@
+
+    # Relationships
+    chef_profile_id = Column(Integer, ForeignKey('profile.id'))
+    chef_profile = relationship('Profile')
+
+    # All fields are required hence nullable=False
+    # General Info -- No length constraints because they
+    # could all end up being fairly long
+    name = Column(String, nullable=False)
+    description = Column(Text, nullable=False)
+    available = Column(Boolean, nullable=False)
+    cuisine = Column(String, nullable=False)
+    price = Column(Integer, nullable=False)
+
+    # These store lists as strings in CSV format
+    ingredients = Column(Text, nullable=False)
+    required_items = Column(Text, nullable=False)
+    image_urls = Column(Text, nullable=False)
+
+    # TODO Validation for cuisine which 
+    # requires a predefined list of cuisines
+    # Which will also allow a select drop box to be used
+    # instead of an input box
+
+    @validates('price')
+    def validate_price(self, key, price):
+        if not isinstance(price, int):
+            raise AssertionError(f"Expected price to be of type integer but got {type(price)}")
+        new_price = price * 100
+        return new_price
+
+    @validates('image_urls')
+    def validate_image_urls(self, key, image_urls):
+        """
+        Make sure we were passed a list
+        Convert it to a string and return it
+        """
+        if not isinstance(image_urls, list):
+            raise AssertionError(f"Expected field 'image_urls' to be of type list but got {type(image_urls)}")
+        stringified_urls = ",".join(image_urls)
+        return stringified_urls
+
+    @validates('ingredients')
+    def validate_ingredients(self, key, ingredients):
+        if not isinstance(ingredients, list):
+            raise AssertionError(f"Expected field 'ingredients' to be of type list but got {type(ingredients)}")
+        stringified_ingredients = ",".join(ingredients)
+        return stringified_ingredients
+
+    @validates('required_items')
+    def validate_required_items(self, key, required_items):
+        if not isinstance(required_items, list):
+            raise AssertionError(f"Expected field 'required_items' to be of type list but got {type(required_items)}")
+        stringified_required_items = ",".join(required_items)
+        return stringified_required_items
+
+    @validates('available')
+    def validate_available(self, key, available):
+        """
+        In case someone sends the value of a checkbox directly
+        from html (which would yield 'true'), make sure it's a 
+        true Boolean.
+        """
+        if not isinstance(available, bool):
+            raise AssertionError(f"Expected field 'available' to be a boolean but got {type(available)}")
+        return available
+
+    @property
+    def get_price(self):
+        """
+        Just in case someone doesn't want the formatted
+        price, give back the stored price divided by 100
+        """
+        return self.price / 100
```

Comment:
I'm not sure this will be necessary. `recipe.price` will already return the integer representation

Reply:
I figured that, since someone would have to divide the price by 100 when getting it, that I would just make a simple method for it.
Should I just delete this?

Reply:
YEah I would just delete it

Reply:
Okay, thanks